Sources/App/Models/Build.swift

```diff
@@ -114,8 +114,17 @@ extension Build {
         case ok
         case failed
         case infrastructureError
-        case pending
+        case triggered
         case timeout
+
+        var isCompleted: Bool {
+            switch self {
+                case .ok, .failed, .timeout:
+                    return true
```

Comment:
NB: this is a small change: timeout wasn't previously considered as "completed" but we actually are done with it in terms of processing, so that's why I put it here.